localstack-core/localstack/services/kinesis/plugins.py

```diff
@@ -1,8 +1,16 @@
+import localstack.config as config
 from localstack.packages import Package, package
 
 
 @package(name="kinesis-mock")
 def kinesismock_package() -> Package:
-    from localstack.services.kinesis.packages import kinesismock_package
+    from localstack.services.kinesis.packages import (
+        KinesisMockEngine,
+        kinesismock_package,
+        kinesismock_scala_package,
+    )
+
+    if KinesisMockEngine(config.KINESIS_MOCK_PROVIDER_ENGINE) == KinesisMockEngine.SCALA:
+        return kinesismock_scala_package
```

Comment:
Q: These package plugins are mostly used for `lpm`, right? Would it make sense to have two different packages here, instead of relying on the environment variable when calling `lpm`?

Reply:
Makes sense, thanks for the clarification! I missed the comment above it seems!